modules/server/src/projects/spinup.js

```diff
@@ -69,14 +69,6 @@ export default ({ app }) => async (req, res) => {
         let fields = await es.search({
           index: `arranger-projects-${id}-${type.index}`,
           type: `arranger-projects-${id}-${type.index}`,
-          size: 0,
```

Comment:
Is this on purpose?

Reply:
This is part of the normalization commit